dandischema/models.py

```diff
@@ -380,10 +176,13 @@ def schema_extra(schema: Dict[str, Any], model) -> None:
                 # In pydantic 1.8+ all Literals are mapped on to enum
                 # This presently breaks the schema editor UI. Revert
                 # to const when generating the schema.
+                # Note: this no longer happens with custom metaclass
                 if prop == "schemaKey":
-                    if len(value["enum"]) == 1:
+                    if "enum" in value and len(value["enum"]) == 1:
                         value["const"] = value["enum"][0]
```

Comment:
absent coverage here confirms that it doesn't happen across CIs either after moving to custom metaclass, hm... good or bad? ;)